common/primitives/src/handles.rs

```diff
@@ -4,6 +4,7 @@ use crate::utils::*;
 use frame_support::BoundedVec;
 use parity_scale_codec::{Decode, Encode};
 use scale_info::TypeInfo;
+#[cfg(feature = "std")]
 use serde::{Deserialize, Serialize};
```

Comment:
Noticed this should have the cfg on it when I used another that needed it removed.